contrib/HACK.md

```diff
@@ -22,14 +22,20 @@ $ docker stack rm func
 $ docker network create func_functions --driver=overlay --attachable=true
 ```
 
-Now edit the `docker-compose.yml` file and add an attribute to the `functions` network of `attachable`.
+Now edit the `docker-compose.yml` file and replace the existing networks block with:
+
+```
+networks:
+    functions:
```

Comment:
So "attachable" isn't needed anymore?

Reply:
Not in the docker-compose.yml file because the network has been created with `docker network create func_functions --driver=overlay --attachable=true`

Reply:
Yes that's right.. I missed the line above in the diff. Cool let's merge it.